src/cdk-experimental/scrolling/virtual-scroll-viewport.ts

```diff
@@ -315,7 +311,7 @@ export class CdkVirtualScrollViewport implements OnInit, OnDestroy {
     this._isChangeDetectionPending = false;
 
     // Apply changes to Angular bindings.
-    this._ngZone.run(() => this._changeDetectorRef.detectChanges());
+    this._ngZone.run(() => this._changeDetectorRef.markForCheck());
```

Comment:
Add a comment here that explains the context around using `markForCheck` so that someone else down the line doesn't try to switch to `detectChanges` again?